docs/api.rst

```diff
@@ -1491,6 +1491,19 @@ Translations
     :<json string operation: Operation to perform: one of ``push``, ``pull``, ``commit``, ``reset``, ``cleanup``
     :>json boolean result: result of the operation
 
+.. http:post:: /api/translations/(string:project)/(string:component)/(string:language)/add_monolingual/
```

Comment:
Thinking about that, I'd call the endpoint just `add`. Right now it's limited to (some) monolingual files, but that might be lifted in the future. ... Or even better, this should be an action on existing `units` endpoint - as this is what it does in the end - it adds new unit.

Reply:
That makes sense. I think I will add it to the existing units endpoint by allowing POST if monolingual files

Reply:
It just needs the permission check. It doesn't work on all monolingual files and it might work on others in future.